app/assets/stylesheets/finder_frontend.scss

```diff
@@ -385,3 +385,7 @@ mark {
   margin-bottom: govuk-spacing(7);
   padding-bottom: govuk-spacing(0);
 }
+
+.govuk-radios {
```

Comment:
the padding is specific to the small radios, but the styling is applied to the generic radios class which will affect the large (normal) radios as well - producing unwanted results, suggest we scope the adjustment to `govuk-radios--small`

Reply:
changed it so that it's only radio smalls

Reply:
I've changed the others so that the movement is all in the `left` attribute - but for radio-smalls I was only able to get it to work with the `padding-left`. Hope that's alright!